Modules/System/Email/src/Email/Outbox/EmailOutbox.Table.al

```diff
@@ -109,4 +109,54 @@ table 8888 "Email Outbox"
         {
         }
     }
+
+    procedure GetMessageId(var MessageId: Guid)
```

Comment:
Isn't `Guid` an allowed return type? I believe it is.

Reply:
Yes, you were right. When I first wrote those getters i didn't got Guid with the Intellisense, but now it looks fine.